workout-tracker/src/components/MainNavBar/SideNavigation/DashboardNavItem.css

```diff
@@ -0,0 +1,70 @@
+.nav-items {
+  padding: 2rem .5rem;
+}
+
+.dashboard-list-item {
+  display: flex;
+  word-spacing: 2em;
+  padding: 15px ;
+  /* list-style-type: none; */
+  text-align: left;
+  font-size: 1.3rem;
+ 
+}
+
+.dashboard-list-item:hover,
+.dashboard-list-item.active,
+.dashboard-list-item:active {
+  background-color: #cce6f4;
+  border-radius: 10px 0 0 10px;
+  border-left: 11px solid #5c0d91;
+}
+
+.dashbaord-list-item a,
+.dashbaord-list-item a:hover {
+  color: #696969;
+  font-weight: bold;
+}
+.dashbaord-list-item a.active {
+  color: #5c0d91;
+}
+.dashboard-list-item a{
+  color:white;
+  word-wrap: none;
+}
+.dashbaord-list-item a:last-of-type {
+  padding-left: 0.75rem;
+}
+.ant-layout-sider-zero-width-trigger {
+
+  background:#0086c9;
+
+}
+.ant-layout-sider-zero-width-trigger:hover {
+  background: #0086c9;
+}
+.ant-layout-sider {
+  background: #0086c9;
+
+}
+.jzzyaV .ant-layout-sider {
+
+  background-color: #0086c9;
+ 
+}
+.dashboard-list-item {
+
+  border-left: 11px solid #0086c9;
+}
+/*  #001529; */
+
+.ant-avatar {
+  color: #fff;
+  background:#5c0d91;
+  width: 72px;
+  height: 72px;
+  border-radius: 30%;
+}
+.ant-layout-header {
+  background:  #0086c9;;
+}
```

Comment:
This CSS file was removed and the styles have been migrated to the DashboardNavItem.js and Layout.js files as styled-components.

Adding this file back creates duplicate styles that might cause our code to behave in unexpected ways. For clarity, the main reason this file was removed was due to lack of specificity with css and ant-design.